extensions/vscode/src/views/homeView.ts

```diff
@@ -849,8 +852,7 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
           (config) =>
             config.configurationName ===
               deploymentObjects.configuration.configurationName &&
-            config.configurationPath ===
-              deploymentObjects.contentRecord.projectDir,
+            config.projectDir === deploymentObjects.configuration.projectDir,
```

Comment:
This appeared to be incorrect and caused issues when debugging where the configuration was never found, stayed `undefined` when doing `getActiveConfig()` and this solved that.